theme.js

```diff
@@ -1544,6 +1544,30 @@ export default (variables = defaultThemeVariables) => ({
       };
     },
   },
+  'shoutem.ui.EmptyListView': {
+    'shoutem.ui.View': {
```

Comment:
if you decide to go for suggested approach from above, this style will be much simpler, and won't depend on component composition, meaning you could reuse it in multiple locations, not just in this specific component sequence